src/fhirpath/search.py

```diff
@@ -388,7 +387,7 @@ def parse_query_string(query_string, allow_none=False):
             parts = q.split("=")
             param_name = unquote_plus(parts[0])
             try:
-                value = parts[1] and unquote_plus(parts[1]) or None
```

Comment:
is unquote_plus still used ?

Reply:
Yep, in `parse_query_string` on the param name